universal-application-tool-0.0.1/app/services/cloud/azure/BlobStorage.java

```diff
@@ -260,9 +260,7 @@ public String getSasToken(String fileName, Optional<String> prefixedOriginalFile
 
     @Override
     public String getBlobUrl(String fileName) {
-      BlobClient blobClient = blobContainerClient.getBlobClient(fileName);
-      String signedUrl = blobClient.getBlobUrl().replace("azurite", "localhost");
-      return signedUrl;
+      return blobContainerClient.getBlobClient(fileName).getBlobUrl();
```

Comment:
fyi this is showing up in the PR

Reply:
This is intentional.

The one azure test we're running right now is broken on main (and this branch) because of the replace here. The replace was accidentally merged into main in a different PR, this fixes the test.